app/controllers/billing_expenses_controller.rb

```diff
@@ -52,6 +45,34 @@ def set_volunteer
     @volunteer = Volunteer.find(params[:volunteer_id]) if params[:volunteer_id]
   end
 
+  def set_volunteer_hours
+    @volunteer_hours = @volunteer.hours.where(billing_expense: nil)
+  end
+
+  def compute_hours
+    return if @volunteer_hours.empty?
+    hours = @volunteer_hours.sum(:hours)
+    minutes = @volunteer_hours.sum(:minutes)
+    hours += minutes / 60
+    compute_amount(hours)
+  end
+
+  def compute_amount(hours)
+    if hours < 25
+      50
+    elsif hours < 50
+      100
+    else
+      150
+    end
+  end
+
+  def volunteer_hours_update
+    @volunteer_hours.each do |hour|
+      hour.update(billing_expense: @billing_expense)
+    end
```

Comment:
`@volunteer_hours.update_all(billing_expense: @billing_expense)`

Reply:
RuboCop complains: Avoid using `update_all` because it skips validations.
Should I skip RuboCop or leave it as it is?

Reply:
Ok, I used ~update_all~ with a comment.

Reply:
Sorry I meant `update`, not `update_all` :) Fixed in bb37bfc